evalml/utils/gen_utils.py

```diff
@@ -511,3 +511,23 @@ def save_plot(fig, filepath=None, format='png', interactive=False, return_filepa
 
     if return_filepath:
         return filepath
+
+
+def deprecate_arg(old_arg, new_arg, old_value, new_value):
```

Comment:
Should we add a test for this helper since it's public?

Reply:
Done!